src/App.js

```diff
@@ -1,5 +1,10 @@
 import React, { useEffect } from 'react';
 
+// Google Analytics
+import ReactGA from 'react-ga';
```

Comment:
Comment (blocking):
We need to wrap this in a if statement, that checks if the environment is production. This can be checked using
```
if(process.env.NODE_ENV === 'production') {
 // do stuff
}
```

then make sure your env.local file has a NODE_ENV property set to development, staging, etc

Reply:
Updated and pushed.